package-lock.json

```diff
@@ -1,8 +1,14697 @@
 {
   "name": "fp-ts-routing",
   "version": "0.5.4",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
@gcanti Because I'm using npm 8.1.2 on my machine, lockfile version here is updated to `2`, is this fine for you? 🙏